build/conf/local.conf

```diff
@@ -0,0 +1,55 @@
+# EdgeOS Build Configuration Template
```

Comment:
did you mean to add this file? usually we generate it dynamically with `./bootstrap.sh` based on a template. mostly because we're actively changing what goes into local.conf.

Reply:
other than that, the rest looks good to me!